app/views/core/CreateAccountModal/CreateAccountModal.coffee

```diff
@@ -94,6 +105,7 @@ module.exports = class CreateAccountModal extends ModalView
     @listenTo @insertSubView(new ChooseAccountTypeView()),
       'choose-path': (path) ->
         if path is 'teacher'
+          startSignupTracking()
           window.tracker?.trackEvent 'Teachers Create Account Loaded', category: 'Teachers' # This is a legacy event name
```

Comment:
Why are we tracking 2 events here? Should we re-use this existing event name - 'Teachers Create Account Loaded' since its also used at https://github.com/codecombat/codecombat/blob/master/app/views/teachers/CreateTeacherAccountView.coffee#L40?

Reply:
We're tracking two events because David wants a single event for create teacher signup started and there are two user flows.
Whilst we still want to keep legacy events. 

And that teachers create account loaded doesn't fire when you are immediately navigated to the create teacher panel.

Reply:
We want to move to unique events for unique actions, which is why we'd have this new event here.  `Teachers Create Account Loaded` is particularly bad since it seems to be using the same event for create account page and modal.

We can leave this legacy event in place for now, since it shouldn't harm anything.  Can remove as future work if it's not being used anymore.

Reply:
Sounds good, thanks for clarifying